codegen/src/main/java/software/amazon/awssdk/codegen/poet/client/AsyncClientClass.java

```diff
@@ -158,8 +158,9 @@ private MethodSpec closeMethod() {
         // For streaming operations we also want to notify the response handler of any exception.
         if (opModel.hasStreamingOutput() || opModel.hasEventStreamOutput()) {
             String paramName = opModel.hasStreamingOutput() ? "asyncResponseTransformer" : "asyncResponseHandler";
+            String errorMethodName = opModel.hasStreamingOutput() ? "onError" : "exceptionOccurred";
```

Comment:
Any reason not to just keep it consistent, even if it's not ideal?

Reply:
No reason, other than I think `onError` is more consistent with the rest of the methods, but I don't feel strongly about it. I'll go ahead and make them consistent.